deploy/layered-sre-authorization/01-layered-cs-sre-admin-cluster.ClusterRole.yaml

```diff
@@ -59,6 +59,3 @@ rules:
   - clusterresourcequotas
   verbs:
   - '*'
-  resourceNames:
-  - rhmi-loadbalancer-quota
-  - rhmi-persistent-volume-quota
```

Comment:
Removing this means CS SRE can manage any clusterresourcequota.  For non-CCS clusters we have service load balancer and storage quotas in place.  Would it work to split out "create" without resourceName limitations and other verbs restricted to these two resources?  I assume these two are the ones that need to be managed.

Reply:
So this will be removed for GA. It is a temporary measure for LA (CCS). RHMI won't be available on non-CCS clusters during LA. I'll add a JIRA to the GA epic to track removing this.

Reply:
Generally would prefer we don't open up this gap but hive will reconcile any changes in core OSD quotas for non-CCS clusters so it will be a mitigated risk.

I'll approve but will ask someone else in SREP to sanity check for a lgtm.